modules/perception/camera/tools/offline/visualizer.cc

```diff
@@ -1058,44 +1096,99 @@ void Visualizer::Draw2Dand3D_all_info_single_camera(
           p_proj[i].y = static_cast<int>(proj[i](1) / proj[i](2));
         }
       }
+      if (object->b_cipv) {
+        cv::line(image_3D, p_proj[0], p_proj[1], color_cipv_,
+          cipv_line_thickness_);
```

Comment:
Align with first parameter.

Reply:
Indented all to be aligned with first parameter